app/controllers/packages_controller.rb

```diff
@@ -217,7 +217,9 @@ def validate_file_ids(file_ids, current_user, errors)
           errors << "file with id '#{file_id}' could not be found"
         else
           data_file = DataFile.find(id_as_int)
-          if data_file.is_authorised_for_access_by?(current_user)
+          if data_file.is_incomplete_package? || data_file.is_error_file?
+            errors << "unable to package error file '#{file_id}'"
```

Comment:
Bad grammar in the error message; it should be something like ''file '#{file_id}' is not in a state that can be packaged"